src/database/migrations/20240104050829_DisableOptOutRole.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('apiRoles', (table) => {
+    table.boolean('disabled').defaultTo(true).notNullable();
+  });
+
+  const allowedRoles = ['MAPPER', 'GENERATOR', 'ID_READER', 'SHARER'];
```

Comment:
perhaps `enabledRoles` makes more sense here?

Reply:
Ah yeah that does work better.